On systems with libtinfo separate from libncurses: src/vimpc-screen.o: undefined reference to symbol 'keypad'

For example on Gentoo with ncurses[tinfo] (without the following patch):
```
g++       -pthread -g -O2 -D__STRICT_ANSI__  -L/opt/local/lib -o vimpc src/vimpc-algorithm.o src/vimpc-buffers.o src/vimpc-clientstate.o src/vimpc-colours.o src/vimpc-errorcodes.o src/vimpc-events.o src/vimpc-mpdclient.o src/vimpc-output.o src/vimpc-player.o src/vimpc-regex.o src/vimpc-screen.o src/vimpc-settings.o src/vimpc-song.o src/vimpc-vimpc.o src/buffer/vimpc-browse.o src/buffer/vimpc-library.o src/buffer/vimpc-directory.o src/mode/vimpc-command.o src/mode/vimpc-inputmode.o src/mode/vimpc-normal.o src/mode/vimpc-search.o src/window/vimpc-browsewindow.o src/window/vimpc-console.o src/window/vimpc-debug.o src/window/vimpc-directorywindow.o src/window/vimpc-error.o src/window/vimpc-help.o src/window/vimpc-infowindow.o src/window/vimpc-librarywindow.o src/window/vimpc-listwindow.o src/window/vimpc-modewindow.o src/window/vimpc-outputwindow.o src/window/vimpc-pagerwindow.o src/window/vimpc-playlistwindow.o src/window/vimpc-scrollwindow.o src/window/vimpc-selectwindow.o src/window/vimpc-songwindow.o src/window/vimpc-window.o src/window/vimpc-windowselector.o src/vimpc-main.o src/vimpc-lyricsloader.o src/vimpc-lyricsfetcher.o src/window/vimpc-lyricswindow.o  -ltag -lpcre -lmpdclient -lcurl  -lncursesw
/usr/lib/gcc/x86_64-pc-linux-gnu/8.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: src/vimpc-screen.o: undefined reference to symbol 'keypad'
/usr/lib/gcc/x86_64-pc-linux-gnu/8.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /lib64/libtinfow.so.6: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make: *** [Makefile:790: vimpc] Error 1
```

There is a patch in Gentoo tree to fix this: https://gitweb.gentoo.org/repo/gentoo.git/tree/media-sound/vimpc/files/vimpc-0.09.1-tinfo.patch?id=63d949769e25176de0d879cbef2b7cd80348ad0f

It would be nice to submit the patch upstream.

--- a/configure.ac
+++ b/configure.ac
@@ -96,6 +96,8 @@ fi
 
 LIBS="$LIBS $CURSES_LIB"
 
+AC_SEARCH_LIBS([keypad], [tinfow tinfo])
+
 PKG_CHECK_MODULES(pcre, libpcre)
 
 AC_SUBST(pcre_LIBS)

